### DISCUSSION TITLE
How to divide a Job on several sub-jobs?

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2451] [General]
I'll give you an example right away.
Technical assistance intervention on an Internet site.
I create in Kimai a Job with a Customer, Project = Site and Activity = Assistance
Total duration 4 hours, during which I however made phone calls to the customer, I took remote control of his PC, I opened assistance tickets with the supplier, I wrote messages in WhatsApp, I sent emails.
Each of these activities have durations of minutes, and I wish I could be counted separately.
Now I am using Tags and so that Job is associated with all these activities. However, I cannot use them for the calculation of times because I would have a 4-hour Job and 6 Tags that Kimai always values them for 4 hours each. Total 28 hours!
How could I handle it?

**Comment by Solidus-Prime** [comment]
We have a similar problem:

We have large presses in the building that can run multiple jobs at the same time. If a worker walks up and places 4 jobs in the machine at the same time and runs them all for 15 mins, the clock logs an entire hour for him (15 mins x 4).

We are experimenting with a "payroll" project. The employees clock into this at the beginning of the day, and clock out at the end to get accurate payroll hours. They then punch into and out of individual jobs as necessary throughout the day to get accurate job-hour reporting.

The only problem with this is that it effectively doubles their hours. We are trying to figure out a way around this now.
